gtsfm/scene_optimizer.py

```diff
@@ -396,6 +404,17 @@ def save_full_frontend_metrics(
                 "inlier_ratio_gt_model": round(report.inlier_ratio_gt_model, PRINT_NUM_SIG_FIGS)
                 if report.inlier_ratio_gt_model
                 else None,
+                "inlier_avg_reproj_error_gt_model": round(
```

Comment:
should we populate these on datasets that do not have the mesh?

Reply:
Isn't it projecting a point onto a line?

Reply:
It's related to it. It's a first order approximation derived using Lagrangian multipliers. I wouldn't call it reprojection error, since it's not a projection of a 3d point into the image, but maybe we can get away with calling it this.